includes/EditPage.php

```diff
@@ -1377,12 +1382,14 @@ function internalAttemptSave( &$result, $bot = false ) {
 			// Run post-section-merge edit filter
 			if ( !wfRunHooks( 'EditFilterMerged', array( $this, $text, &$this->hookError, $this->summary ) ) ) {
 				# Error messages etc. could be handled within the hook...
+				$result['hookaborted'] = 'EditFilterMerged'; // Wikia change - SUS-1188
 				$status->fatal( 'hookaborted' );
 				$status->value = self::AS_HOOK_ERROR;
 				wfProfileOut( __METHOD__ );
 				return $status;
 			} elseif ( $this->hookError != '' ) {
 				# ...or the hook could be expecting us to produce an error
+				$result['hookaborted'] = 'EditFilterMerged'; // Wikia change - SUS-1188
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)